wagtail/admin/tests/pages/test_reorder_page.py

```diff
@@ -0,0 +1,112 @@
+from unittest import mock
+
+from django.contrib.auth import get_user_model
+from django.contrib.auth.models import Group, Permission
+from django.contrib.messages import constants as message_constants
+from django.http import HttpRequest, HttpResponse
+from django.test import TestCase
+from django.urls import reverse
+
+from wagtail.admin.navigation import get_explorable_root_page
+from wagtail.core.models import Page
+from wagtail.core.signals import post_page_move, pre_page_move
+from wagtail.tests.testapp.models import BusinessChild, BusinessIndex, SimplePage
+from wagtail.tests.utils import WagtailTestUtils
+
+
+class TestPageReorder(TestCase, WagtailTestUtils):
+    fixtures = ["test.json"]
+
+    def setUp(self):
+        # Find root page
+        self.root_page = Page.objects.get(id=2)
+
+        # Create a simple page tree:
+        # root
+        # |- index_page (BusinessIndex)
+        #    |- child_1 (BusinessChild)
+        #    |- child_2 (BusinessChild)
+        #    |- child_3 (BusinessChild)
```

Comment:
So enhance this tree with a SimplePage with two SimplePage's as childs and test them (they should already work without fixing the bug for the business pages.